### ISSUE TITLE
[BUG] Oraxen breaks Vanilla Note Block and Observer behavior

### ISSUE THREAD

**Issue by ChristianHine** [oraxen/oraxen#1075]
### I have checked...

- [X] I am using the latest version of Oraxen
- [X] I am using the latest version of any dependencies
- [X] I am using ProtocolLib 5.0
- [X] I have checked if any similar bug reports exist

### Description

After installing Oraxen observers stopped to see state changes (Right-Clicking to Note Block) in Note Blocks 

### Steps to reproduce

1) Placing Note Block in front of Observer
2) Right-Click to Note Block

### Plugin Version

Oraxen version 1.162.1

### ProtocolLib Version

ProtocolLib version 5.1.0

### Server Software & Version

Current: git-Purpur-2062 (MC: 1.20.1)

### How breaking is the bug?

Minor Bug - Plugin completely functional, but features have non-working aspects

**Comment by Boy0000** [comment]
This isnt a bug but a side-effect of custom blocks. If you dont want that simply disable noteblock menchanic in mechanics.yml

**Comment by ryantheleach** [comment]
Is it possible to mitigate the side effects? 

Oraxen could implement a custom noteblock, that records the data of the block somewhere other than blockstates, and trigger all the required redstone.

I know you have it as not-planned, but it would both let people use the extra blockstates, AND still have something resembling a noteblock still in the game.

**Comment by Boy0000** [comment]
I intend only to reimplement it so that normal interactions work.
I do not intend to reimplement it so you have full redstone functionality.

Is it possible? Probably.
Is it worth it? No i dont believe it is.

In the year+ ive worked on the plugin I hve had at worst a handful of people who wanted full redstone functionality AND custom blocks. It just is not a realistic good solution for it imo.
Rather make requests erc to Mojang to add CB so we dont need to do this scuffed workaround
Because thats what it is, a workaround, its not intended to be a feature with 0 side effects

**Comment by ryantheleach** [comment]
The issue is, is that server admins are willing to make the sacrifice, but players who are forced into that situation often have their builds break when the admin installs the plugin.

**Comment by Boy0000** [comment]
Mojang has actively been making changes and new features that aims to make servers able to do more "custom" stuff
There is also some internal code that hints at it in registries etc
like multiple server-resourcepacks in 1.20.3 & Display Entities, Interaction Entities without culling, custom biomes etc
It is not unrealistic to think that they will implement it some day

Tldr dont expect this to come to Oraxen. It just is a feature the vast majority is fine with loosing if it means custom blocks.
And it is not a simple thing to reimplement without some trade-off that frankly isnt worth it
